src/pageObject.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
-class Header {
+const f = 'Conduit logo';
+const y = '[Publish comment] button';
 
+class Header {
+  get logo() {
+    return f;
+  };
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the ' + `${f}`;
+  };
 };
 
 class ArticlePage extends PageObject {
```

Comment:
article page doesn't contain constructor. ArticlePage constructor should takes url and slug. Constructor should modify original url by adding slug at the end